### ISSUE TITLE
helper functions get_id and get_url are in the documentation but not in the helpers module in notion-client 0.4.0

### ISSUE THREAD

**Issue by Thompsonmina** [ramnes/notion-sdk-py#64]
I inspected the source for the notion client i installed and only the pick function is contained in the helpers module even though the documentation says that get_id and get_url are supposed to be included

**Comment by ramnes** [comment]
Yeah, we should probably deploy the documentation of the latest released version and the one of the current git head in two different places. PR welcome!

**Comment by ghost** [comment]
Pinging this. Docs are still behind latest version.

**Comment by ramnes** [comment]
What? The issue here is the exact opposite, i.e. the documentation is always ahead of the latest version, not behind. Did you see something that's actually released but not part of the documentation?

**Comment by wangdong2023** [comment]
Hi, I'm new here. For me the document is missing as [here](https://github.com/ramnes/notion-sdk-py/tree/main/docs/reference), where can I find more docs?

**Comment by ramnes** [comment]
https://ramnes.github.io/notion-sdk-py/

**Comment by nat-n** [comment]
> Yeah, we should probably deploy the documentation of the latest released version and the one of the current git head in two different places. PR welcome!

I'd suggest two alternate approaches that might be neater than managing multiple documentation deployments:
1. maintain a development branch separate from main. So documentation changes for existing features or urgent bug fixes can merge straight to main, but everything else merges to the development branch, which is only merged to main as part of the release process.
2. Enforce conventional commits to main and [automate creating a new release](https://blog.openreplay.com/automating-releases-in-github-with-conventional-commits/) for every functional change.

WDYT?

**Comment by ramnes** [comment]
Do you suggest the two together specifically or one or the other works in your mind?

I'm not too keen on conventional commits, but I don't really mind a `develop` branch. Actually, I used to have one in many of my projects at some point, but with the advent of CI/CD, I found it to be a useless step. Why do you think it would be neater?

Implementing versions is a very common use case, and thus should be quite straightforward. Material for MkDocs even has a documentation page about it: https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/

**Comment by ramnes** [comment]
Do you suggest the two together specifically or one or the other works in your mind?

I'm not too keen on conventional commits, but I don't really mind a develop branch. Actually, I used to have one in many of my projects at some point, but with the advent of CI/CD, I found it to be a useless step, so I'm not too sure I want to come back to such a setup.

I do agree that deploying in two different places is probably overkill thought. I just took a quick look, and implementing versions within the docs is a very common use case. Material for MkDocs even has a documentation page about it: https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/

**Comment by ramnes** [comment]
Finally had a bit of time for this, so the docs now use mike! You'll see the version switch at the top, and the docs now show the latest version documentation by default, not the development docs. Thanks for raising the issue in the first place. :)
